src/Microsoft.Health.Fhir.SchemaManager/CommandCollectionExtensions.cs

```diff
@@ -79,10 +79,12 @@ public static IServiceCollection SetCommandLineOptions(this IServiceCollection s
             string? authenticationTypeValue = config[OptionAliases.AuthenticationType];
             if (!string.IsNullOrWhiteSpace(authenticationTypeValue))
             {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
For these parts of the code using obsolete components, are there new components to replace them? Is there a user-story to work on replacing these components?

Reply:
Good point, created work item here https://microsofthealth.visualstudio.com/Health/_boards/board/t/Olympus/Stories/?workitem=109734